src/current/_includes/releases/release-downloads-docker-image.md

```diff
@@ -74,14 +74,15 @@ This patch release has been withdrawn{% if include.advisory_key %} due to [this
   {% if release.docker.docker_arm == true %}
 [Multi-platform images](https://docs.docker.com/build/building/multi-platform/) include support for both Intel and ARM. Multi-platform images do not take up additional space on your Docker host.
 
-    {% if r.docker.docker_arm_experimental == true %}
```

Comment:
wondering if it's simpler (even if dumber) to leave this as an if/endif and just add more if/endif blocks instead of revising it to be a multi-way conditional

Reply:
I changed it because there were instances where both of the fields were set to true and that caused both texts to show up.